pkgs/by-name/co/cosh/package.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, unstableGitUpdater
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cosh";
+  version = "unstable-2024-02-23";
```

```suggestion
  version = "0.1-unstable-2024-02-23";
```

Also see https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming

Please also change the commit message and PR title.

---

this is unstableGitUpdater's fault, and will be fixed by #280501

---

Please update the version as per the guidelines.